hpi-schul-cloud/schulcloud-server
# BC-6099 add token for deleting data

# Description

Adding api_token checkup for tldraw-document management controller. This controller don't have jwt token authorization and  it is not visible for outside resources (ingress is not configured, as it was planned). But after consultations with architects we should have some sort of safe mechanizm in case of further development.

QA note:
 Change tested on env:
https://bc-6099-add-token-for-deleting-data.dbc.dbildungscloud.dev/
No issues found. Change tested by creating a tldraw table by different users. Checking the table contents in the database. After them deleting the table and deleting the whole course and checking the database.
See : https://ticketsystem.dbildungscloud.de/browse/BC-6099



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-6099


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
